src/crimson/osd/backfill_facades.h

```diff
@@ -35,6 +35,10 @@ struct BackfillState::PeeringFacade {
                                              const pg_stat_t &stats) {
     return peering_state.update_complete_backfill_object_stats(hoid, stats);
   }
+
```

Comment:
Update the commit title. It should be `crimson/osd: settle BackfillState in PGRecovery`.